contracts/ERC5018.sol

```diff
@@ -161,4 +161,19 @@ contract ERC5018 is IERC5018, LargeStorageManager, BlobStorageManager {
         }
         return 0;
     }
+
+    function getChunkHashes(bytes memory name, uint256[] memory chunkIds) public override view returns (bytes32[] memory hashes) {
```

Comment:
This function can only optimize cases where a file has many chunks, but it is less effective for situations involving many small files, where each file has only one or two chunks. I’m curious about the case with Vitalik’s blog—what’s the bottleneck there? Do you have any early testing results?

Reply:
https://github.com/ethstorage/evm-large-storage/pull/11#issue-2491199607 Comparable data added

Reply:
Is it possible to obtain multiple chunk hashes for multiple files at once?

Reply:
The contract is OK, but the changes to ethstorage-sdk will be huge, and the upload interface needs to be changed from a single file to multiple files.